src/Explorer/Controls/SmartUi/SmartUiComponent.tsx

```diff
@@ -67,7 +68,8 @@ interface ChoiceInput extends BaseInput {
 }
 
 interface DescriptionDisplay extends BaseDisplay {
-  description: Description;
+  description?: Description;
+  isDynamicDescription?: boolean;
```

Comment:
>isDynamicDescription [](start = 2, length = 20)

what do you mean by dynamic description? is it description which is computed as in cost for sqlx?

Reply:
yes, it can be set like and of the other controls